installer/README.md

```diff
@@ -2,105 +2,63 @@
 This is a standalone/non-containerized installer for SODA Infrastructure Manager (delfin) project.
 It contains a script and options to check the environment feasible for installing delfin. Installs required dependent software/binaries.
 
-# Prerequisite
-Python3 and Pip3 should be installed on the system.
-Ensure the logged-in user has root privileges.
-
-Note: If you don't have python3 in your system, you may follow below steps to setup python3 virtual environment.
-
-##### How to setup python3 virtual environment and delfin project
-
-1. Install python-virtual environment package and python3-dev package.
-
-  ```sh
-  apt-get install python-virtualenv
-  apt-get install python3-dev
-  ```
-2. Clone delfin repo
-
-  ```sh
-  git clone https://github.com/sodafoundation/delfin.git
-
-  # Example
-  root@root1:~/delfin-demo$ git clone https://github.com/sodafoundation/delfin.git
-
-  root@root1:~/delfin-demo$ cd delfin
-  ```
+# Supported OS
+Ubuntu 16.04, Ubuntu 18.04
 
-3. Create a project, using python3
-  ```sh
-  virtualenv -p /usr/bin/python3.6 project_delfin
+# Prerequisite
 
-  # Example
-  root@root1:~/delfin-demo/delfin$ virtualenv -p /usr/bin/python3.6 project_delfin
-  ```
-4. Activate your project
+- #### Ensure the logged-in user has root privileges.
 
-  ```sh
-  source project_delfin/bin/activate
+- ##### Setup Python3
+    Python3 and Pip3 should be installed on the system.
 
-  # Example
-  root@root1:~/delfin-demo/delfin$ source project_delfin/bin/activate
-  # (project_delfin) root@root1:~/delfin-demo/delfin$
-  ```
-5. Install all requirements using Pip
+    Note: If you don't have python3 in your system, you may follow below steps to setup python3 environment.
 
-  ```sh
-  pip install -r requirements.txt
+    ```sh
+    apt-get install python3
+    apt-get install python3-pip
+    ```
 
-  # Example
-  (project_delfin) root@root1:~/delfin-demo/delfin$ pip install -r requirements.txt
-  ```
-6. set PYTHONPATH to working directory
+- #### Set PYTHONPATH to working directory
 
-  ```sh
-  export PYTHONPATH=$(pwd)
-  ```
-  
+    ```sh
+    export PYTHONPATH=$(pwd)
+    ```
 
-#### How to setup prometheus to monitor performance metric
+- ##### Setup Prometheus (for monitor performance metric)
```

Comment:
Extra '#' here. Formatting may be improved in some places. Please check with 'vew files'

Reply:
Removed extra '#' as per comment